website/docs/docs/dbt-cloud-apis/access-metadata-api.md

```diff
@@ -0,0 +1,63 @@
+---
+title: "Access the Metadata API"
+id: "access-metadata-api"
+---
+
+Use the Metadata API to evaluate data health in the long-term or at a moment-in-time. This section will explain how to access and query the Metadata API and what you'll need.
+
+<Snippet src="metadata-api-prerequisites" />
+
+We provide a [GraphQL explorer](https://metadata.cloud.getdbt.com/graphql) for this API where you can run ad-hoc queries or browse the schema. As GraphQL provides a self-describing API, the schema shown in the GraphiQL interface is an accurate representation of the graph and fields available to query. 
+
+1. Create a [service account token](/docs/dbt-cloud-apis/service-tokens) to authorize requests. dbt Cloud Admin users can generate a _Metadata Only_ service token, which can be used to execute a specific query against the Metadata API for authorization of requests.
+
+2. Access the API by using the endpoint `https://metadata.{YOUR_ACCESS_URL}/graphql`. 
+    * Replace `{YOUR_ACCESS_URL}` with the appropriate [Access URL](/docs/deploy/regions-ip-addresses) for your region and plan. For example, if your multi-tenant region is North America, your endpoint is `https://metadata.cloud.getdbt.com/graphql`. If your multi-tenant region is EMEA, your endpoint is `https://metadata.emea.dbt.com/graphql`.
+
+3. Go to the [GraphQL API](https://metadata.cloud.getdbt.com/graphql) sandbox, which supports arbitrary queries. 
+
+4. To begin querying the Metadata API, refer to the [query documentation](/docs/dbt-cloud-apis/metadata-querying.md) for details on how to use GraphiQL.
+
+5. For specific query points, refer to the [schema documentation](/docs/dbt-cloud-apis/metadata-schema-model). 
+
+
+## Retention limits
+You can use the metadata API to query data from the previous three months. For example, if today was April 1st, you could query data back to January 1st.
```

Comment:
For the purpose of specificity, is it actually held for 3 months or 90 days? The language is often used interchangeably but we should note exactly how long data is held (months vary in length).

Reply:
@cafzal can you confirm if users can query data from the previous 3 months or 90 days?

Reply:
3 months